README.md

```diff
@@ -28,7 +28,6 @@ This repository contains the source for the following packages:
 - **[`@ni/nimble-tokens`](/packages/nimble-tokens/)** - design tokens used by the component packages
 
 And some additional utility packages:
-- [`@ni/beachball-lock-update`](/packages/beachball-lock-update/) - script to update `package-lock.json` files in npm workspace monorepos
 - [`@ni/xliff-to-json-converter`](/packages/xliff-to-json-converter/) - a utility to convert translation files from XLIFF to JSON for Angular localization
```

Comment:
The other task mentioned in the linked issue is to mark the package as deprecated on NPM. What's the plan for staging that?

Reply:
In a perfect world the deprecation would happen first so that we're never in a state where there's an "active" package with no repo backing it. But given the fact we've already stopped using this package it probably doesn't matter practically.